apps/agent-provisioning/AFJ/scripts/fargate.sh

```
@@ -552,50 +312,52 @@
-n=0
-max_attempts=15
-sleep_time=10
-AGENT_HEALTHCHECK_URL="http://$ADMIN_ALB_DNS/agent"
-echo "--------AGENT_HEALTHCHECK_URL-----$AGENT_URL"
-until [ "$n" -ge "$max_attempts" ]; do
-    agentResponse=$(curl -s -o /dev/null -w "%{http_code}" "$AGENT_HEALTHCHECK_URL")
-    if [ "$agentResponse" = "200" ]; then
-        echo "Agent is running"
-        break
-    else
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)